### PR TITLE
Fix actions row component

### PR DESCRIPTION
**Pull request by benjaminpkane**
Fixes a bug resulting from #3603. e2e will confirm the fix

### CODE REVIEW

File: app/packages/core/src/components/ColorModal/ColorFooter.tsx [hunk 1]
```diff
@@ -54,24 +50,9 @@ const ColorFooter: React.FC = () => {
         >
           Reset
         </Button>
-        <Button
-          title={
-            canEdit
```
Comment (imanjra) [inline app/packages/core/src/components/ColorModal/ColorFooter.tsx:59]: This is the newer change and the one we want. Please see https://github.com/voxel51/fiftyone/pull/3573

File: app/packages/core/src/components/ColorModal/ColorFooter.tsx [hunk 2]
```diff
@@ -54,24 +50,9 @@ const ColorFooter: React.FC = () => {
...
-          disabled={!canEdit}
-        >
-          Save as default
-        </Button>
-        {hasSavedSettings && (
+        {canEdit && (
           <Button
-            text={"Save as default"}
-            title={`Save to dataset appConfig`}
+            title={`Save to dataset app config`}
```
Comment (imanjra) [inline app/packages/core/src/components/ColorModal/ColorFooter.tsx:55]: Want to keep this as well along with 57-71

File: app/packages/core/src/components/ColorModal/ColorFooter.tsx [hunk 3]
```diff
@@ -90,21 +71,20 @@ const ColorFooter: React.FC = () => {
...
-            title={canEdit ? "Clear" : "Can not clear in read-only mode"}
+            title={"Clear dataset default"}
             onClick={() => {
               updateDatasetColorScheme(null);
               setDatasetColorScheme({
                 variables: { subscription, datasetName, colorScheme: null },
               });
               setColorScheme({ fields: [], colorPool: defaultColorPool });
             }}
-            disabled={!canEdit}
```
Comment (imanjra) [inline app/packages/core/src/components/ColorModal/ColorFooter.tsx:107]: Need to keep this

File: app/packages/core/src/components/ColorModal/ColorFooter.tsx [hunk 4]
```diff
@@ -90,21 +71,20 @@ const ColorFooter: React.FC = () => {
...
-            style={LONG_BUTTON_STYLE}
-          />
+          >
+            Save as default
+          </Button>
         )}
         {canEdit && datasetDefault && (
           <Button
-            text={"Clear default"}
-            title={canEdit ? "Clear" : "Can not clear in read-only mode"}
```
Comment (imanjra) [inline app/packages/core/src/components/ColorModal/ColorFooter.tsx:99]: Need to keep this

**Review by imanjra — CHANGES_REQUESTED** [review]

**Comment by benjaminpkane** [comment]
Ok 👍 comments addressed

**Review by imanjra — APPROVED** [review]
LGTM! Thank you! 💥

**Pull request merged**